src/awkward/contents/listoffsetarray.py

```diff
@@ -383,20 +383,22 @@ def _broadcast_tooffsets64(self, offsets: Index) -> ListOffsetArray:
             )
 
         # Check whether we need to slice the content, shift our offsets
-        start = self._offsets[0]
+        this_start = self._offsets[0]
         this_zero_offsets = self._offsets.data
-        if index_nplike.known_data and start == 0:
+        if index_nplike.known_data and this_start == 0:
             next_content = self._content
         else:
-            this_zero_offsets = this_zero_offsets - start
-            next_content = self._content[start:]
+            this_zero_offsets = this_zero_offsets - this_start
+            next_content = self._content[this_start:]
```

Comment:
It looks to me like this only renames a variable.

Reply:
Yes!